### ISSUE TITLE
Different reports about installed GuestAdditions version

### ISSUE THREAD

**Issue by hongaar** [ByteInternet/hypernode-vagrant#47]
On `vagrant up`, I always get the following warnings, followed by a reboot, making starting the VM unnecessary slow:

```
==> hypernode: Machine booted and ready!
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   4.2.0
VBoxService inside the vm claims: 5.0.14
Going on, assuming VBoxService is correct...
GuestAdditions seems to be installed (5.0.14) correctly, but not running.
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   4.2.0
VBoxService inside the vm claims: 5.0.14
Going on, assuming VBoxService is correct...
stdin: is not a tty
Usage: service < option > | --status-all | [ service_name [ command | --full-restart ] ]
bash: line 4: vboxadd: command not found
Got different reports about installed GuestAdditions version:
Virtualbox on your host claims:   4.2.0
VBoxService inside the vm claims: 5.0.14
Going on, assuming VBoxService is correct...
stdin: is not a tty
Removing existing VirtualBox DKMS kernel modules ...done.
Removing existing VirtualBox non-DKMS kernel modules ...done.
Building the VirtualBox Guest Additions kernel modules ...done.
Doing non-kernel setup of the Guest Additions ...done.
You should restart your guest to make sure the new modules are actually used

Restarting VM to apply changes...
==> hypernode: Attempting graceful shutdown of VM...
```

This might be related to [this issue](https://github.com/dotless-de/vagrant-vbguest/issues/111) with a proposed solution [here](https://github.com/dotless-de/vagrant-vbguest/issues/111#issuecomment-36723828)

**Comment by vdloo** [comment]
Looks like nobody in that thread has a better solution than editing the ```~/.vagrant.d/boxes/hypernode/2692/virtualbox/box.ovf``` file unfortunately. You could do that but you'd have to keep doing it every release. Instead you could also try to set ```config.vbguest.auto_update``` to False after running the initial vagrant up, but that's also far from ideal. 

I think the best solution for now would be if we just bake the most recent Guest Additions version into the box, which is something we should do anyway. I'll post back here when we've added that to our build process.

**Comment by vdloo** [comment]
hypernode-vagrant box versions => 3085 have Virtualbox 5 guest additions. The build servers are now using the ```deb http://download.virtualbox.org/virtualbox/debian jessie contrib``` repo to keep up with the latest version of Virtualbox, so from now on recent guest additions will be packaged into the box. 

I updated the .ovf file manually to 5.0.24 so the ```Got different reports about installed GuestAdditions version``` warning is gone for now, but as soon as a new vbox version comes out it will return because [vagrant-vbguest](https://github.com/dotless-de/vagrant-vbguest) does not update the .ovf file in packaged boxes. Luckily the warning is harmless because the version in the guest is always leading:

```
VBoxService inside the vm claims: 5.0.24
Going on, assuming VBoxService is correct...
[hypernode] GuestAdditions 5.0.24 running --- OK.
```

so to summarize: if you run `vagrant up` now with the latest version of Virtualbox installed you don't need to wait for the latest guest additions to be compiled anymore because they are included in the box
